.github/workflows/ubuntu.yml

```diff
@@ -0,0 +1,101 @@
+name: Ubuntu (latest)
+
+on:
+  push:
+    branches:
+      - ci_badges
```

Comment:
`ci_badges` is the name of this branch, which I would expect would be deleted after merge. Should the condition to run be pushing to `develop` instead?

Reply:
Yes, this is for debugging purpose. I am checking the pipeline on GitHub Actions.